src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity/Coin.php

```diff
@@ -129,4 +130,17 @@ public function merge(Coin $coin)
         $this->eula = is_null($coin->getEula()) ? null : $coin->getEula();
         $this->oidcClient = is_null($coin->getOidcClient()) ? null : $coin->getOidcClient();
     }
+
+    public function asArray(): array
+    {
+        return [
+            'metaDataFields.coin:application_url' => $this->getApplicationUrl(),
+            'metaDataFields.coin:eula' => $this->getEula(),
+            'metaDataFields.coin:exclude_from_push' => $this->getExcludeFromPush(),
+            'metaDataFields.coin:oidc_client' => $this->getOidcClient(),
+            'metaDataFields.coin:original_metadata_url' => $this->getOriginalMetadataUrl(),
+//            'metaDataFields.coin:service_team_id' => $this->getServiceTeamId(),
+//            'metaDataFields.coin:signature_method' => $this->getSignatureMethod(),
```

Comment:
Commented code should be deleted?

Reply:
Thanks @johanib, for the sake of convenience, I'm fixing this in the successive PR that is already in the making.